addons/web/static/src/legacy/utils.js

```diff
@@ -270,7 +270,14 @@ export function makeLegacyNotificationService(legacyEnv) {
                 }
             }
 
-            legacyEnv.services.notification = { notify, close };
+            function add(message, { title, type }) {
+                notification.add(message, {
+                    title,
+                    type,
+                });
+            }
+
+            legacyEnv.services.notification = { notify, close, add };
```

Comment:
Ok to kind of mix both legacy and new APIs on the legacy version of the notification service. Diff could be simplified though:
```suggestion
            legacyEnv.services.notification = { notify, close, add: notification.add };
```
Note that we are currently removing those legacy services, so hopefully all this won't exist in 17.0